app/lib/onboarding/pages/page.dart

```diff
@@ -0,0 +1,267 @@
+      color: Color(0xFF359600),
+      child: BottomCard(
+        icon: Icon(Icons.warning_rounded, size: 32),
+        getText: (context) => context.l10n.onboarding_3_disclaimer,
+      ),
+    ),
+    OnboardingSubPage(
+      illustrationPath: 'assets/images/onboarding/4.gif',
+      getHeader: (context) => context.l10n.onboarding_4_header,
+      getText: (context) => context.l10n.onboarding_4_text,
+      color: Color(0xFF00B9FA),
+      child: BottomCard(
+        getText: (context) => context.l10n.onboarding_4_button,
+        onClick: () => launchUrl(
+          Uri.parse(
+            'https://www.cdc.gov/genomics/gtesting/genetic_testing.htm',
+          ),
+        ),
+      ),
+    ),
+    OnboardingSubPage(
+      illustrationPath: 'assets/images/onboarding/5.gif',
+      getHeader: (context) => context.l10n.onboarding_5_header,
+      getText: (context) => context.l10n.onboarding_5_text,
+      color: Color(0xFF0A64BC),
+    ),
+  ];
+
+  final _isLoggedIn = MetaData.instance.isLoggedIn ?? false;
+
+  @override
+  Widget build(BuildContext context) {
+    final colors = _pages.map((page) => page.color);
+    final tweenSequenceItems = <TweenSequenceItem>[];
+    for (var tweenIndex = 0; tweenIndex < colors.length - 1; tweenIndex++) {
+      tweenSequenceItems.add(
+        TweenSequenceItem(
+          weight: 1,
+          tween: ColorTween(
+            begin: colors.elementAt(tweenIndex),
+            end: colors.elementAt(tweenIndex + 1),
+          ),
+        ),
+      );
+    }
+    final background = TweenSequence(tweenSequenceItems);
+
+    final pageController = usePageController(initialPage: 0);
+    final currentPage = useState(0);
+
+    return Scaffold(
+      body: AnimatedBuilder(
+        animation: pageController,
+        builder: (context, child) {
+          final color = pageController.hasClients
+              ? pageController.page! / (_pages.length - 1)
+              : .0;
+
+          return DecoratedBox(
+            decoration: BoxDecoration(
+              color: background.evaluate(AlwaysStoppedAnimation(color)),
+            ),
+            child: child,
+          );
+        },
+        child: Stack(alignment: Alignment.topCenter, children: [
+          Positioned.fill(
+            child: PageView(
+              controller: pageController,
+              onPageChanged: (newPage) => currentPage.value = newPage,
+              children: _pages,
+            ),
+          ),
+          Positioned(
+            bottom: 80,
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: _buildPageIndicator(context, currentPage.value),
+            ),
+          ),
+          Positioned(
+            bottom: 16,
+            right: 16,
+            child: _buildNextButton(
+              context,
+              pageController,
+              currentPage.value == _pages.length - 1,
+            ),
+          ),
+        ]),
```

Comment:
```suggestion
        child: Stack(
        alignment: Alignment.topCenter,
        children: [
          Positioned.fill(
            child: PageView(
              controller: pageController,
              onPageChanged: (newPage) => currentPage.value = newPage,
              children: _pages,
            ),
          ),
          Positioned(
            bottom: 80,
            child: Row(
              mainAxisAlignment: MainAxisAlignment.center,
              children: _buildPageIndicator(context, currentPage.value),
            ),
          ),
          Positioned(
            bottom: 16,
            right: 16,
            child: _buildNextButton(
              context,
              pageController,
              currentPage.value == _pages.length - 1,
            ),
          ),
        ],
      ),
```
Trailing comma in the end